extensions/map-layers-formats/package.json

```diff
@@ -47,20 +47,19 @@
     "@types/chai-as-promised": "^7",
     "@types/geojson": "~7946.0.14",
     "@types/google-protobuf": "~3.15.6",
+    "@types/jsdom": "^21.1.7",
     "@types/mocha": "^10.0.6",
     "@types/sinon": "^17.0.2",
     "@types/sinon-chai": "^3.2.0",
     "chai": "^4.3.10",
     "chai-as-promised": "^7.1.1",
-    "cpx2": "^3.0.0",
+    "cpx2": "^8.0.0",
     "eslint": "^9.13.0",
-    "ignore-styles": "^5.0.1",
-    "jsdom": "^19.0.0",
-    "jsdom-global": "3.0.2",
```

Comment:
Don't know if you seen but there is an up to date alternative called [`global-jsdom`](https://www.npmjs.com/package/global-jsdom). 😄

Reply:
That's still [going agains jsdom's best practices](https://github.com/jsdom/jsdom/wiki/Don%27t-stuff-jsdom-globals-onto-the-Node-global), no?

Reply:
I am not sure, just suggested a drop in replacement for deprecated package.

Looking over react testing library docs they suggest using `global-jsdom` when running with mocha. https://testing-library.com/docs/dom-testing-library/setup/#using-without-jest
I would guess that other test runners/frameworks does something similar too.

Reply:
I realize a lot of test runners do something similar, but I think the `jsdom` docs make a good point that you wind up with a strange hybrid franken-environment that doesn't strictly match either a real-world node or browser runtime.

@hl662 would these tests be a good candidate for running in `vitest-browser` maybe?

Reply:
I would inspect the tests and see how they use the browser APIs before deciding if we need to move to using vitest-browser - I'm not too familiar with this package so I can't say if using a real browser would be better than mocking the browser APIs out.

Reply:
In my opinion until test aren't updated to run with vitest browser there is nothing wrong with using global-jsdom. At least we would use standart way to polute node globals instead of our own. 😄